tests/jballerina-unit-test/src/test/resources/test-src/documentation/markdown_negative.bal

```diff
@@ -110,3 +110,8 @@ final string testConsts = "TestConstantDocumentation";
 public function load(string filePath) {
 
 }
+
+# + a - parameter a
```

Comment:
Even the following wouldn't work, right?

```ballerina
# Docs.
# + a - docs for a
# + b - docs for b
[int, float] [a, b] = getTuple();
```

IMO, we need to have a way to document the individual variables. Shall we initiate a discussion regarding this?

Reply:
+1